spryker/usr/local/share/spryker/spryker_functions.sh

```diff
@@ -157,6 +157,17 @@ do_spryker_run_tests() {
   as_code_owner "vendor/bin/codecept run --skip Acceptance"
 }
 
+do_spryker_hosts() {
+    set +e
+    grep -q zed /etc/hosts
+    HOSTS_UPDATED=$?
+    set -e
+    if [ "$HOSTS_UPDATED" != "0" ]; then
+        # zed is required for internal endpoint and others are required for Acceptance tests to not loop over internet in order to resolve those endpoints
+        echo "127.0.0.1 zed $ZED_HOST $YVES_HOST" >> /etc/hosts
```

Comment:
This should not be mandatory. Some platforms refuse to write in this file.

Reply:
so instead we could just use the k8s dns for CP and use docker-compose extra_hosts/link aliases

Reply:
hmm, actually, can't do the k8s dns way for when on the same host, as it's set for the multiple vhosts, and k8s dns is one dns address per service I guess